lib/flatten.js

```diff
@@ -14,6 +14,9 @@ function FlattenSourceMap(map) {
   });
 
   originalMap._mappings.forEach(function(m){
+    if (mapping.length < 5) {
```

Comment:
I assume you're trying to find the case where `m` does not have 5 members (`[gline, gcol, source, oline, ocol]`). While heavy handed, you could solve your immediate issue by checking `m.length`, rather than `mapping.length` (mapping is the actual object, and defined below).

A better approach would be to use default values when referencing from `m` when creating the mapping object; unfortunately, it's been some time since I looked at this code and I don't remember exactly what format `m`'s members should be in.

Either way, this will generate incorrect mappings when the incoming mapping is out of whack. So on second thought, maybe ignoring bad mappings *is* a good idea...